changelog/unreleased/4470

```diff
@@ -0,0 +1,6 @@
+Enhancement: Talkback announces the view label correctly
```

Comment:
```suggestion
Enhancement: TalkBack announces the view label correctly
```
You wrote it correctly in the description but not in the title

Reply:
Done